pkgs/development/python-modules/torchrl/default.nix

```diff
@@ -132,46 +133,51 @@ buildPythonPackage rec {
     ++ optional-dependencies.gym-continuous
     ++ optional-dependencies.rendering;
 
-  disabledTests = [
-    # torchrl is incompatible with gymnasium>=1.0
-    # https://github.com/pytorch/rl/discussions/2483
-    "test_resetting_strategies"
-    "test_torchrl_to_gym"
-
-    # mujoco.FatalError: an OpenGL platform library has not been loaded into this process, this most likely means that a valid OpenGL context has not been created before mjr_makeContext was called
-    "test_vecenvs_env"
-
-    # ValueError: Can't write images with one color channel.
-    "test_log_video"
-
-    # Those tests require the ALE environments (provided by unpackaged shimmy)
-    "test_collector_env_reset"
-    "test_gym"
-    "test_gym_fake_td"
-    "test_recorder"
-    "test_recorder_load"
-    "test_rollout"
-    "test_parallel_trans_env_check"
-    "test_serial_trans_env_check"
-    "test_single_trans_env_check"
-    "test_td_creation_from_spec"
-    "test_trans_parallel_env_check"
-    "test_trans_serial_env_check"
-    "test_transform_env"
-
-    # undeterministic
-    "test_distributed_collector_updatepolicy"
-    "test_timeit"
-
-    # On a 24 threads system
-    # assert torch.get_num_threads() == max(1, init_threads - 3)
-    # AssertionError: assert 23 == 21
-    "test_auto_num_threads"
-
-    # Flaky (hangs indefinitely on some CPUs)
-    "test_gae_multidim"
-    "test_gae_param_as_tensor"
-  ];
+  disabledTests =
+    [
+      # torchrl is incompatible with gymnasium>=1.0
+      # https://github.com/pytorch/rl/discussions/2483
+      "test_resetting_strategies"
+      "test_torchrl_to_gym"
+
+      # mujoco.FatalError: an OpenGL platform library has not been loaded into this process, this most likely means that a valid OpenGL context has not been created before mjr_makeContext was called
+      "test_vecenvs_env"
+
+      # ValueError: Can't write images with one color channel.
+      "test_log_video"
+
+      # Those tests require the ALE environments (provided by unpackaged shimmy)
+      "test_collector_env_reset"
+      "test_gym"
+      "test_gym_fake_td"
+      "test_recorder"
+      "test_recorder_load"
+      "test_rollout"
+      "test_parallel_trans_env_check"
+      "test_serial_trans_env_check"
+      "test_single_trans_env_check"
+      "test_td_creation_from_spec"
+      "test_trans_parallel_env_check"
+      "test_trans_serial_env_check"
+      "test_transform_env"
+
+      # undeterministic
+      "test_distributed_collector_updatepolicy"
+      "test_timeit"
+
+      # On a 24 threads system
+      # assert torch.get_num_threads() == max(1, init_threads - 3)
+      # AssertionError: assert 23 == 21
+      "test_auto_num_threads"
+
+      # Flaky (hangs indefinitely on some CPUs)
+      "test_gae_multidim"
+      "test_gae_param_as_tensor"
+    ]
+    ++ lib.optionals (stdenv.hostPlatform.isLinux && stdenv.hostPlatform.isAarch64) [
```

Comment:
The new formatter is broken and no one cares....

Reply:
I raised that issue months ago with that exact change pattern and was ignored.

Reply:
https://discourse.nixos.org/t/formatting-team-meeting-2024-12-10-and-2025-01-07/58466